examples/src/examples/user-interface/text-typewriter/example.mjs

```diff
@@ -1,3 +1,4 @@
+// @flag WEBGPU_ENABLED
```

Comment:
are you using some existing `@flag` functionality? If not, could we give it some better name, perhaps `@example` or something?

Reply:
No this is custom and the word flag is just arbitrary just to signify its a config flag option so can be called anything